EventsExpress/ClientApp/src/components/helpers/helpers.js

```diff
@@ -49,6 +50,17 @@ export const validate = values => {
         }
     });
 
+    occurenceFields.forEach(field => {
```

Comment:
Why do you need to chaeck it here and then the second time on line 99. I think the correct one is on line 99. Please remove this one.